Description: disable building plugins requiring non-DFSG compliant 3rdparty code

Author: Lu YaNing <dluyaning@gmail.com>
Last-Update: 2022-10-14


--- a/src/plugins/CMakeLists.txt
+++ b/src/plugins/CMakeLists.txt
@@ -11,18 +11,6 @@ if(QT_FEATURE_hunspell)
     add_subdirectory(hunspell)
     list(APPEND plugins_imports QtQuick.VirtualKeyboard.Plugins.Hunspell/auto)
 endif()
-if(QT_FEATURE_openwnn)
-    add_subdirectory(openwnn)
-    list(APPEND plugins_imports QtQuick.VirtualKeyboard.Plugins.OpenWNN/auto)
-endif()
-if(QT_FEATURE_pinyin AND NOT QT_FEATURE_cerence_xt9)
-    add_subdirectory(pinyin)
-    list(APPEND plugins_imports QtQuick.VirtualKeyboard.Plugins.Pinyin/auto)
-endif()
-if(QT_FEATURE_tcime)
-    add_subdirectory(tcime)
-    list(APPEND plugins_imports QtQuick.VirtualKeyboard.Plugins.TCIme/auto)
-endif()
 if(QT_FEATURE_myscript)
     add_subdirectory(myscript)
     list(APPEND plugins_imports QtQuick.VirtualKeyboard.Plugins.MyScript/auto)
diff --git a/src/plugins/openwnn/CMakeLists.txt b/src/plugins/openwnn/CMakeLists.txt
index 26c2c6a1..668a369e 100644
--- a/src/plugins/openwnn/CMakeLists.txt
+++ b/src/plugins/openwnn/CMakeLists.txt
@@ -1,5 +1,3 @@
-add_subdirectory(3rdparty/openwnn)
-
 #####################################################################
 ## QtVirtualKeyboardOpenWnnPlugin Plugin:
 #####################################################################
diff --git a/src/plugins/pinyin/CMakeLists.txt b/src/plugins/pinyin/CMakeLists.txt
index ebeb04f4..0ce7b42a 100644
--- a/src/plugins/pinyin/CMakeLists.txt
+++ b/src/plugins/pinyin/CMakeLists.txt
@@ -1,5 +1,3 @@
-add_subdirectory(3rdparty/pinyin)
-
 #####################################################################
 ## QtVirtualKeyboardPinyinPlugin Plugin:
 #####################################################################
diff --git a/src/plugins/tcime/CMakeLists.txt b/src/plugins/tcime/CMakeLists.txt
index e6d076fe..38d03781 100644
--- a/src/plugins/tcime/CMakeLists.txt
+++ b/src/plugins/tcime/CMakeLists.txt
@@ -1,5 +1,3 @@
-add_subdirectory(3rdparty/tcime)
-
 #####################################################################
 ## QtVirtualKeyboardTCImePlugin Plugin:
 #####################################################################

